io_export_cryblend/export.py

```diff
@@ -745,20 +745,30 @@ def __process_bone_weights(self, object_, armature, skin_node):
         vw = ""
         vertex_groups_lengths = ""
         vertex_count = 0
+        vertex_group_count = 0
+        bone_list = {}
+
+        for bone_id, bone in enumerate(bones):
+            bone_list[bone.name] = bone_id;
 
         for vertex in object_.data.vertices:
+            vertex_group_count = 0
             for group in vertex.groups:
+                group_name = object_.vertex_groups[group.group].name
+                if (group.weight == 0 or
+                    group_name not in bone_list):
+                    continue

```

Comment:
Ignore groups with zero weight and also groups not associated to any bone (group name not matching any bone name)